flask/tests/conftest.py

```diff
@@ -37,6 +38,14 @@ def application():
     yield test_app
 
 
+def add_json_content_header(kwargs: dict):
+    """helper method for injecting content-type: json into headers"""
+    headers = kwargs.pop("headers", {})
+    headers["Content-Type"] = "application/json"
+    kwargs["headers"] = headers
+    return kwargs
+
+
```

Comment:
I think this is no longer used now as well

Reply:
fixed in [75f8c38](https://github.com/ccmbioinfo/stager/pull/484/commits/75f8c38e245b8d2ffed64bd849c9df2c948638af).